src/view/components/decisionScopes.jsx

```diff
@@ -77,104 +77,122 @@ const validationSchema = object().shape({
-                                isQuiet
-                                isDisabled={decisionScopesArray.length === 1}
-                                variant="secondary"
-                                onPress={() => {
-                                  arrayHelpers.remove(index);
-                                }}
-                                aria-label="Remove decision scope"
-                                UNSAFE_className="u-verticalCenterBottom"
-                                minWidth={0}
-                              >
-                                <Delete />
-                              </Button>
-                            </div>
-                          );
-                        })}
+                      <div className="u-gapBottom" key={index}>
+                        <TextField
+                          data-test-id={`scope${index}Field`}
+                          name={`decisionScopesArray.${index}`}
+                          width="size-5000"
+                          aria-label="Decision scope"
+                        />
                         <Button
+                          data-test-id={`deleteScope${index}Button`}
+                          isQuiet
+                          isDisabled={decisionScopesArray.length === 1}
                           variant="secondary"
-                          data-test-id="addDecisionScopeButton"
                           onPress={() => {
-                            arrayHelpers.push("");
+                            arrayHelpers.remove(index);
                           }}
+                          aria-label="Remove decision scope"
+                          UNSAFE_className="u-verticalCenterBottom"
+                          minWidth={0}
                         >
-                          Add scope
+                          <Delete />
                         </Button>
                       </div>
                     );
-                  }}
-                />
-              </div>
-            )}
-          </Fragment>
-        );
+                  })}
+                  <Button
+                    variant="secondary"
+                    data-test-id="addDecisionScopeButton"
+                    onPress={() => {
+                      arrayHelpers.push("");
+                    }}
+                  >
+                    Add scope
+                  </Button>
+                </div>
+              );
+            }}
+          />
+        </div>
+      )}
+    </Fragment>
+  );
+};
+
+DecisionScopes.propTypes = {
+  initInfo: PropTypes.object,
+  formikProps: PropTypes.object,
+  registerImperativeFormApi: PropTypes.func
+};
+
+const DecisionScopesExtensionView = () => {
+  return (
+    <ExtensionViewForm
+      render={props => {
+        return <DecisionScopes {...props} />;
       }}
     />
   );
 };
 
-export default DecisionScopes;
+export default DecisionScopesExtensionView;
```

Comment:
The changes to DecisionScopes were just to accommodate the refactored ExtensionViewForm.